python/sidecar/http.py

```diff
@@ -0,0 +1,50 @@
+import sidecar
+from fastapi import FastAPI
+from sidecar._typing import (RewriteMannerType, RewriteRequest,
+                             RewriteResponse, TextCompletionRequest,
+                             TextCompletionResponse)
+
+app = FastAPI()
+
+
+@app.get("/")
+async def index():
+    return {"message": "Hello World"}
+
+
+@app.get("/rewrite")
+async def rewrite(
+    text: str,
+    manner: RewriteMannerType = "concise",
+    n_choices: int = 1,
+    temperature: float = 0.7
+) -> RewriteResponse:
+    request = RewriteRequest(
```

Comment:
Is there any way to get FastAPI to give you the request body as a `RewriteRequest` object?

Reply:
Yep! I've got that working on a different branch now.